fetch.bs

```diff
@@ -4692,7 +4719,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    <li><p>If <var>response</var> is not a
    <a>network error</a> and <var>request</var>'s
    <a for=request>cache mode</a> is not "<code>no-store</code>",
-   update <var>response</var> in the HTTP cache for <var>request</var>.
+   update <var>response</var> in the <a>HTTP cache</a> for <var>request</var>.
```

Comment:
This should be `<var>httpCache</var>`, and not the general definition of a HTTP cache, right?

Reply:
done